### ISSUE TITLE
Feature Request: Allow to associate a work item with the checkin

### ISSUE THREAD

**Issue by ControlTheBit** [ivangabriele/vscode-tfs#38]
This is one of the most important features for us to work efficiently with this extension. It would be really nice to have this included in some future release. Thank you!

![image](https://cloud.githubusercontent.com/assets/9163917/25609040/b6af2f1c-2f1d-11e7-9816-d365b02793bd.png)

**Comment by dflynn1024** [comment]
Yes, I second that.. Is there away currently to override check-in policies such as requiring a work item id?
